bees/emailbee/emailbee.go

```diff
@@ -43,18 +45,40 @@ func (mod *EmailBee) Action(action bees.Action) []bees.Placeholder {
 
 	switch action.Name {
 	case "send":
-		to := ""
-		text := ""
-		subject := ""
-
+		var to, mailText, subject string
+		var addOn bool
 		action.Options.Bind("recipient", &to)
-		action.Options.Bind("text", &text)
+		action.Options.Bind("text", &mailText)
 		action.Options.Bind("subject", &subject)
+		action.Options.Bind("HTML", &addOn)
+
+		var host string
+		var port int
+		if strings.Index(mod.server, ":") != -1 {
+			host = strings.Split(mod.server, ":")[0]
+			port, _ = strconv.Atoi(strings.Split(mod.server, ":")[1])
+		} else {
+			host = mod.server
+			port = 587
+		}
+
+		re, _ := regexp.Compile("\\<[\\S\\s]+?\\>")
```

Comment:
Don't try to manually strip the tags. Use "text" as the plaintext content and "html" as the text/html content of the mail.